python/lsst/ts/xml/generate_subsystems_doc.py

```diff
@@ -291,12 +291,19 @@ def generate_subsystems_doc():
                             write_heading(cf, name="Enumerations")
                             enumeration_seen = True
                         states = topic.text.split(",")
+                        print(states)
                         state_name = states[0].split("_")[0].lstrip()
                         cf.write(f":{state_name}:\n")
                         for state in states:
+                            state = state.split("_")[1:]
+                            state = "".join(state)
+                            state = state.lstrip()
```

Comment:
Is this line actually needed? This is all the text after the first underscore, so I'm puzzled as to what you can be stripping out here. (That said: the comment I requested earlier explaining the format might clarify).